tests/e2e/utils/utils.js

```diff
@@ -295,6 +295,38 @@ function extractWorkbenchData(data) {
   return workbenchData;
 }
 
+function getGrayLogSnapshotUrl(target_url, since_secs = 30) {
+  let monitoring_base_url;
+  let snapshot_url;
+
+  // WARNING: This mappings might change
+  const table = {
+    "staging.osparc.io": "https://monitoring.staging.osparc.io/graylog/",
+    "osparc.io": "https://monitoring.osparc.io/graylog/",
+    "osparc-master.speag.com": "https://monitoring.osparc-master.speag.com/graylog/",
+    "osparc-staging.speag.com": "https://monitoring.osparc.speag.com/graylog/",
+    "osparc.speag.com": "https://monitoring.osparc.speag.com/graylog/",
+  };
+  for (var key in table) {
+    if (target_url.startsWith("https://" + key)) {
+      monitoring_base_url = table[key];
+      break;
+    }
+  }
```

Comment:
This could be done:
```
const { hostname } = new URL(targetUrl);
monitoringBaseUrl = table[hostname] || null;

if (monitoringBaseUrl) {
  ...
```

Reply:
highly appreciated by a javascript beginner like me ! :-)